apps/src/javalab/JavalabView.jsx

```diff
@@ -33,13 +35,21 @@ class JavalabView extends React.Component {
     channelId: PropTypes.string
   };
 
-  state = {
-    isRunning: false,
-    isTesting: false
-  };
+  constructor(props) {
+    super(props);
+
+    this.setHeaderHeight = this.setHeaderHeight.bind(this);
```

Comment:
this isn't needed because `setHeaderHeight` is an arrow function, so it's automatically bound to `this`. that also means that you can remove the constructor and set the initial state in the same way it was previously

Reply:
Oh awesome- thanks. Done!